src/cityreader/cityreader.py

```diff
@@ -20,14 +25,30 @@ def cityreader(cities=[]):
   # TODO Implement the functionality to read from the 'cities.csv' file
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
-    return cities
+  import csv
+  
+  for item in csv.DictReader(open('cities.csv', 'r')):
```

Comment:
Good find with csv.DictReader. It's best practice to use a `with open() as f:` code block to access file contents but most likely it won't cause issues in this case. https://askubuntu.com/questions/701491/is-closing-a-file-after-having-opened-it-with-open-required-in-python